content/docs/command-reference/plots/diff.md

```diff
@@ -6,13 +6,14 @@ plotting them in a single image.
 ## Synopsis
 
 ```usage
-usage: dvc plots diff [-h] [-q | -v] [-t [TEMPLATE]] [-d [DATAFILE]] [-f FILE]
-                     [-s SELECT] [-x X] [-y Y] [--stdout] [--no-csv-header]
-                     [--no-html] [--title TITLE] [--xlab XLAB] [--ylab YLAB]
-                     [revisions [revisions ...]]
+usage: dvc plots diff [-h] [-q | -v] [-t <path>]
+                      [--targets [<path> [<path> ...]]] [-o <path>]
+                      [-x <field>] [-y <field>] [--no-csv-header]
+                      [--show-vega] [--title <text>] [--xlab <text>]
+                      [--ylab <text>] [revisions [revisions ...]]
```

Comment:
`--xlab` -> `--x-label` and `--ylab` -> `--y-label`

Reply:
Just noticed these. Yes, I updated all these. Please also notice this: https://github.com/iterative/dvc/pull/3948#issuecomment-638470590 (option argument names should match this PR).